### ISSUE TITLE
[Question] [Dolphin] Printing plaintext passwords in process environment variables

### ISSUE THREAD

**Issue by licanxue** [apache/dolphinscheduler#16628]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

Printing plaintext passwords in process environment variables

### What you expected to happen

no passwords showing in process environment variables

### Version

dev

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by SbloodyS** [comment]
Can you elaborate your issue? I don't quite understand what this is.

**Comment by licanxue** [comment]
We used DolphinScheduler as a scheduling tool in our project, and during our security testing we found that we executed the ‘ps -ef‘ command to query Dolphin's process on the machine where DolphinScheduler was installed, and then we viewed the process by the command ‘cat /proc/procId/environ‘ environment variable information, in which the password is shown in clear text, it is determined to be a security issue, may I ask how to fix it?

**Comment by SbloodyS** [comment]
I don't think this is a cve since most open source tools store the account password in plain text in the service's configuration file.
